docs/source/troubleshooting/submit_debug_info.rst

```diff
@@ -81,3 +81,125 @@ and uploaded to StackStorm.
     2015-02-10 17:43:49,016  INFO - Collecting files...
     2015-02-10 17:43:49,770  INFO - Creating tarball...
     2015-02-10 17:43:49,912  INFO - Debug tarball successfully generated and can be reviewed at: /tmp/st2-debug-output-vagrant-ubuntu-trusty-64-2015-02-10-17:43:49.tar.gz
+
+
+Submitting debugging information to Plexxi
+==========================================
+
+The enhancement work will allow submit_debug_info.py to be customized for specific deployments (Plexxi's as an example :-) by loading a set of overrides from a YAML file. The following config options can now be specified:
+
+* log_file_paths - an additional set of log files to gather
+* conf_file_paths - an additional set of config files to gather
+* s3_bucket - the S3 bucket to upload the archive to
+* gpg - gpg key and fingerprint to use when encrypting the archive
+* shell_commands - a list of shell commands to execute and capture the output from
+* company_name - the company name to show in the interactive prompts and log messages
+
+Sample Config yaml file:
+
+log_file_paths:
```

Comment:
use .. code-block:: yaml or .. sourcecode:: yaml before the yaml block and indent everything yaml with 1tab/4spaces.